### ISSUE TITLE
Changes to imported CSS files inside subfolders of frontend/styles/ do not trigger a rebuild/refresh

### ISSUE THREAD

**Issue by nolder3193** [bridgetownrb/bridgetown#535]
I have a components folder nested directly inside the styles folder containing an index.css file that imports individual component CSS files within the same folder (`/frontend/styles/components/index.css`).  That `index.css` file then gets imported into the main `index.css` file, one level up (`/frontend/styles/index.css`).  When making changes inside nav.css (`/frontend/styles/components/nav.css`), for example, saving the file does not trigger a refresh/rebuild unless I manually restart the server or save a file outside of that folder—excluding `/frontend/styles/components/index.css` as any change in that file **_does_** trigger a rebuild.

<img width="217" alt="CleanShot 2022-04-14 at 14 25 33@2x" src="https://user-images.githubusercontent.com/32823514/163453569-7163d741-ecef-448e-bee6-4a0b3afdab05.png">

**Bridgetown Version**:

bridgetown 1.0.0 "Pearl"

**To Reproduce**

- Generate a new bridgetown repo at v1.0.0. Set up Tailwind CSS using bundled configuration.
- Ensure `postcss-import ^14.0.2` npm package is installed.
- Add `'postcss-import': {}` and `'tailwindcss/nesting': {}` under `module.exports.plugins` in `postcss.config.js`.
- Create a subfolder named `components` nested directly inside `/frontend/styles/`.
- Create a file named `index.css` and `nav.css` inside that folder (`nav` can be renamed to anything).
- Inside `/frontend/styles/components/index.css` add the following line: `@import './nav.css';`.
- Inside the main CSS index file, `/frontend/styles/index.css`, change the contents of the file to the following:
```
@import 'jit-refresh.css'; /* triggers frontend rebuilds */

/* Base/global styles */
@import 'tailwindcss/base';

/* Components */
@import 'tailwindcss/components';
@import './components/index.css';
@import 'tailwindcss/utilities';
```

**Current behavior**

Changes made to `/frontend/styles/components/nav.css` do not trigger a rebuild upon saving the file, despite its import being resolved successfully.

**Expected behavior**

Changes made to `/frontend/styles/components/nav.css` should trigger a rebuild upon saving the file, provided its import is resolved successfully.

**Screenshots**

![CleanShot 2022-04-14 at 10 19 22@2x](https://user-images.githubusercontent.com/32823514/163453101-d3a131d0-5226-4ef9-8970-c19ddd39631a.png)
![CleanShot 2022-04-14 at 10 20 54@2x](https://user-images.githubusercontent.com/32823514/163453103-d947e361-e8c1-441a-a3a8-4b0d1e4d1bb8.png)
![CleanShot 2022-04-14 at 10 21 56@2x](https://user-images.githubusercontent.com/32823514/163453106-f11ccd84-f582-4978-a477-f7aa25a9133c.png)

**Computing environment (please complete the following information):**
 - OS: MacOS 12.2 Monterey
 - Browser: Google Chrome 
 - Browser Version: 100.0.4896.88 (Official Build) (x86_64)
 - Ruby Version: ruby 2.7.4p191 (2021-07-07 revision a21a3b7d23) [x86_64-darwin21]

**Additional context**

**Comment by jaredcwhite** [comment]
@nolder3193 I confirmed the issue, and it turns out to be a bug in our default esbuild config. I'll be getting a fix into the Bridgetown 1.1 beta release shortly. Thanks for the report.

**Comment by jaredcwhite** [comment]
Resolved in #542
